doc/ref_arch/kubernetes/chapters/chapter02.rst

```diff
@@ -211,6 +217,9 @@ Cloud Infrastructure Software Profile Capabilities
 
 **(2)** In Kubernetes based infrastructures packet monitoring is out of the scope for the infrastructure.
 
+**(3)** Host operating system must provide drivers etc. to support listed capabilities. More information,
```

Comment:
can be moved to third column of `i.os.001`

Reply:
done